external/radiation/python/getdata.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
+from config import *
+import os 
+import xarray as xr 
+import warnings
+
+
+def random_numbers(LOOKUP_DIR, me):
```

Comment:
nit: can you use lower case for the function arguments in this file. Typically we reserve all caps for module level constants. Also some type hints would be helpful since it isn't clear what "me" is.
```suggestion
def random_numbers(lookup_dir: str, me: int):
```